src/particle/pdgid/functions.py

```diff
@@ -444,6 +471,20 @@ def is_composite_quark_or_lepton(pdgid):
     return _digit(pdgid, Location.N) == 4 and _digit(pdgid, Location.Nr) == 0
 
 
+@_deprecated(
+    version="0.16.0",
+    reason="This method is deprecated and will be removed from version 0.17.0. Use is_excited_quark_or_lepton instead.",
```

Comment:
```suggestion
    reason="This method will be removed from version 0.17.0. Use is_excited_quark_or_lepton instead.",
```

This already states "deprecated" in the warning, no need to duplicate.